Source/WebCore/animation/WebAnimationUtilities.cpp

```diff
@@ -107,6 +107,18 @@ static bool compareStyleOriginatedAnimationOwningElementPositionsInDocumentTreeO
     auto& bReferenceElement = b.element;
 
     if (&aReferenceElement == &bReferenceElement) {
+        if (isNamedViewTransitionPseudoElement(a.pseudoElementIdentifier) && isNamedViewTransitionPseudoElement(b.pseudoElementIdentifier) && a.pseudoElementIdentifier->nameArgument != b.pseudoElementIdentifier->nameArgument) {
+            RefPtr activeViewTransition = aReferenceElement.document().activeViewTransition();
+            ASSERT(activeViewTransition);
+            for (auto& key : activeViewTransition->namedElements().keys()) {
+                if (key == a.pseudoElementIdentifier->nameArgument)
+                    return true;
+                if (key == b.pseudoElementIdentifier->nameArgument)
+                    return false;
+            }
+            return false;
```

Comment:
I don't understand this `return false`.

Can we actually reach that point (or should it be an `ASSERT_NOT_REACHED()`? ) ? If so, why `false` and not `true` ?

Reply:
ASSERT_NOT_REACHED() would work too, the spec treats unknown names as added at the end of the named elements map so false is correct I believe.